### ISSUE TITLE
Issue on building image with hhvm

### ISSUE THREAD

**Issue by krzysztof-madej** [actions/runner-images#7783]
### Description

Building VM image a test fails because hhvm haven't been installed

```
    azure-arm: Err:11 https://dl.hhvm.com/ubuntu focal Release
    azure-arm:   Certificate verification failed: The certificate is NOT trusted. The certificate issuer is unknown.  Could not handshake: Error in the certificate verification. [IP: 13.226.153.127 443]
    azure-arm: Reading package lists...
==> azure-arm: E: The repository 'https://dl.hhvm.com/ubuntu focal Release' does not have a Release file.
==> azure-arm: E: Unable to locate package hhvm

```

```
    azure-arm: Running tests from '/imagegeneration/tests/Tools.Tests.ps1'
    azure-arm: Describing HHVM
    azure-arm:   [-] hhvm 128ms (99ms|30ms)
    azure-arm:    Command 'hhvm --version' has finished with exit code
    azure-arm:        /usr/bin/bash: hhvm: command not found
    azure-arm:    at "hhvm --version" | Should -ReturnZeroExitCode, /imagegeneration/tests/Tools.Tests.ps1:265
    azure-arm:    at <ScriptBlock>, /imagegeneration/tests/Tools.Tests.ps1:265
    azure-arm: Tests completed in 867ms
    azure-arm: Tests Passed: 0, Failed: 1, Skipped: 0 NotRun: 98
```

### Platforms affected

- [X] Azure DevOps
- [ ] GitHub Actions - Standard Runners
- [ ] GitHub Actions - Larger Runners

### Runner images affected

- [X] Ubuntu 20.04
- [ ] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [ ] macOS 13
- [ ] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

It's not the runner image.

### Is it regression?

yes

### Expected behavior

An image for ubuntu 20.04 should be possible to build.

### Actual behavior

I cannot build an image for the build agent based on ubuntu 20.04

### Repro steps

1. Build an image

**Comment by krzysztof-madej** [comment]
The same is happening here https://github.com/actions/runner-images/actions/runs/5372485127/jobs/9745919564

**Comment by erik-bershel** [comment]
Hey @krzysztof-madej! We'll take a look.

**Comment by erik-bershel** [comment]
@krzysztof-madej upstream repo key expired. We'll work on possible fix on our side.

**Comment by erik-bershel** [comment]
Waiting for an answer here: https://github.com/facebook/hhvm/issues/9376

**Comment by mikhailkoliada** [comment]
Should be bypassed in main in the meantime

**Comment by wsmelton** [comment]
If we are building based on image tags this still fails. Will there be a release soon for Ubuntu OS tags?

**Comment by erik-bershel** [comment]
@wsmelton hey!
A stable build is only possible from the latest state of main branch.
I do not recommend using release tags for building - this will not help to build a completely similar image due to the principle of writing scripts and will only lead to problems with the build.

**Comment by wsmelton** [comment]
@erik-bershel That seems like it contradicts how images are tagged for latest by this repository if that were true. If I go build a Windows Server 2022 runner from tip of main right now it will not match the current `windows-latest` image will it (as of July 3)?

**Comment by erik-bershel** [comment]
@wsmelton yep. We never promise that with our sources you'll build the same image as hosted runners. A lot of scripts install "latest" version of software. Tags is used to create connection between exact image version and code used to create that image. But there never was a connection like to "build the same image from the tag's code".

**Comment by htekdev** [comment]
Is this related to PR:
https://github.com/actions/runner-images/pull/7817

**Comment by erik-bershel** [comment]
@htekdev yes. It is a temp workaround for issue with Meta certificate.
